### PR TITLE
Add a few missing type annotations

### PR DESCRIPTION
**Pull request by Rigidity**
### Purpose:

Adds type annotations that were missing in a few places, which mypy (configured more strictly for my editor, presumably) was complaining about.

### CODE REVIEW

**Review by AmineKhaldi — DISMISSED** [review]

File: chia/util/struct_stream.py [hunk 1]
```diff
@@ -89,4 +89,4 @@ def stream_to_bytes(self) -> bytes:
 
     # this is meant to avoid mixing up construcing a bytes object of a specific
     # size (i.e. bytes(int)) vs. serializing the integer to bytes (i.e. bytes(uint32))
-    __bytes__ = None
+    __bytes__: None = None
```
Comment (altendky) [inline chia/util/struct_stream.py:92]: This was changed in release and then checkpointed so presumably it is just conflicted.  It's a regular function def again (for now) so probably just revert this.

```suggestion
    __bytes__ = None
```

**Review by altendky — CHANGES_REQUESTED** [review]
I would guess your IDE isn't using mypy.  If it were and was using the strict config or such it'd be complaining a lot more.  But, I'm just guessing.

File: chia/util/keyring_wrapper.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ class KeyringWrapper:
     """
 
     # Static members
-    __shared_instance = None
+    __shared_instance: Optional[KeyringWrapper] = None
     __keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH
```
Comment (altendky) [inline chia/util/keyring_wrapper.py:67]:
```suggestion
    __shared_instance: ClassVar[Optional[KeyringWrapper]] = None
    __keys_root_path: ClassVar[Path] = DEFAULT_KEYS_ROOT_PATH
```

**Comment by Rigidity** [comment]
> I would guess your IDE isn't using mypy. If it were and was using the strict config or such it'd be complaining a lot more. But, I'm just guessing.

It should be, don't know why it didn't catch this. Do I need to configure it to lint more strictly?

**Review by altendky — DISMISSED** [review]

**Review by altendky — APPROVED** [review]

**Review by arvidn — APPROVED** [review]

**Pull request merged**